### ISSUE TITLE
Public Ansible Project Meeting Agenda - April 2020

### ISSUE THREAD

**Issue by jillr** [ansible/community#532]
Please leave a comment regarding any agenda item you wish to discuss. If you don't show up for the meeting, your item will be skipped. If your IRC nick is different from your Github username, leave that as well.

See https://github.com/ansible/community/blob/master/meetings/README.md for the schedule

Once an item has been discussed it should get checked off.

If you just want reviewers for your contribution try the #ansible-devel irc channel on freenode.

**Comment by s-hertel** [comment]
Meeting ended Thu Apr  2 15:16:05 2020 UTC. Information about MeetBot at http://wiki.debian.org/MeetBot .
11:16 Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2020-04-02/ansible_core_public_irc_meeting_https:github.comansiblecommunityissues532.2020-04-02-15.01.html
11:16 Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2020-04-02/ansible_core_public_irc_meeting_https:github.comansiblecommunityissues532.2020-04-02-15.01.txt
11:16 Log: https://meetbot.fedoraproject.org/ansible-meeting/2020-04-02/ansible_core_public_irc_meeting_https:github.comansiblecommunityissues532.2020-04-02-15.01.log.html

**Comment by DaazKu** [comment]
Hi, 

I would like to bring attention on this [PR](https://github.com/ansible/ansible/pull/68288) at tomorrow's meeting.
It is a "simple enough" code change based on an existing proposal that allows better roles dependency/requirement management. The documentation about it, if the change is accepted, will probable be the harder part to nail down.

**Comment by bcoca** [comment]
Meeting ended Thu Apr  9 15:32:33 2020 UTC. Information about MeetBot at http://wiki.debian.org/MeetBot .
Minutes: https://meetbot.fedoraproject.org/ansible-meeting/2020-04-09/ansible_core_plublic_irc_meeting.2020-04-09-15.08.html
Minutes (text): https://meetbot.fedoraproject.org/ansible-meeting/2020-04-09/ansible_core_plublic_irc_meeting.2020-04-09-15.08.txt
Log: https://meetbot.fedoraproject.org/ansible-meeting/2020-04-09/ansible_core_plublic_irc_meeting.2020-04-09-15.08.log.html

**Comment by cfelder** [comment]
Hi I had a discussion with agaffney on freenode #ansible. He suggested to add this to the core meeting agenda and show up in the meeting. I've a meeting scheduled for 1h - 1h before but I'll try to show up tomorrow, April 23.

- [ ] It would be nice to reuse existing roles also within a collection, e.g. for roles where no collection exists. agaffney pointed out that collections are considered self-contained. Nevertheless it would not make sense to duplicate roles not managed by oneself. Therefore it would be nice to reuse roles installed with ``ansible-galaxy`` e.g. to ``~/.ansible/roles`` when using collections. Currently this leads to the following error message, e.g.

> ERROR! the role 'podman_container_systemd' was not found in my_namespace.my_collection:ansible.builtin

**Comment by geerlingguy** [comment]
@cfelder (just FYI this might be related: https://github.com/ansible/ansible/issues/69101)

**Comment by ericsysmin** [comment]
I also noticed that modules within those roles when called will not work as expected either.

**Comment by cfelder** [comment]
@geerlingguy Thanks for pointing that our. I also tested the PR ansible/ansible#69105 which is a Backport to stable-2.9 which works fine for me. Meanwhile I can workaround using an additional 
```
collections:
  - ansible.legacy
```
inside my collection where I am including a standalone rule using
```
include_role:
  name: podman_container_systemd
```

Regarding this topic there is nothing to discuss for me.

I've also a PR which I've already created on January and rebased from time to time where I would be glad to get some comments but I am uncertain this meeting is the correct platform. Just for reference ansible/ansible#66570
